tests/_support/Helper/ManifestHelper.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdkTest\Helper;
+
+use Codeception\Module;
+use Generated\Shared\Transfer\ManifestRequestTransfer;
+use Generated\Shared\Transfer\ManifestResponseTransfer;
+use Generated\Shared\Transfer\ManifestTransfer;
+use org\bovigo\vfs\vfsStream;
+use SprykerSdk\Zed\AppSdk\AppSdkConfig;
+use SprykerTest\Shared\Testify\Helper\ConfigHelperTrait;
+use SprykerTest\Zed\Testify\Helper\Business\BusinessHelperTrait;
+
+class ManifestHelper extends Module
+{
+    use BusinessHelperTrait;
+    use ConfigHelperTrait;
+
+    /**
+     * @var string|null
+     */
+    protected ?string $rootUrl = null;
+
+    /**
+     * @return \Generated\Shared\Transfer\ManifestRequestTransfer
+     */
+    public function haveManifestCreateRequest(): ManifestRequestTransfer
+    {
+        $this->getValidatorHelper()->mockRoot($this->getRootUrl());
+
+        $config = $this->getValidatorHelper()->getConfig() ?? new AppSdkConfig();
+
+        $manifestTransfer = new ManifestTransfer();
+        $manifestTransfer
+            ->setName('Manifest')
+            ->setLocaleName('en_US');
+
+        $manifestRequestTransfer = new ManifestRequestTransfer();
+        $manifestRequestTransfer
+            ->setManifestPath($config->getDefaultManifestPath())
+            ->setManifest($manifestTransfer);
+
+        return $manifestRequestTransfer;
+    }
+
+    /**
+     * @return \Generated\Shared\Transfer\ManifestTransfer
+     */
+    public function haveManifestCreateTransfer(): ManifestTransfer
```

Comment:
Why having 2 methods `haveManifestCreateTransfer`? Probably you were on the way to make this one returning an invalid?